src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { FilterOption } from '../../types/FilterOption';
+
+interface Props {
+  todos: Todo[];
+  filter: string;
+  setFilter: (filter: FilterOption) => void;
+}
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filter,
+  setFilter,
+}) => {
+  const completedTodos = todos.filter((todo) => todo.completed);
+  const hasCompletedTodo = completedTodos.length > 0;
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">{`${completedTodos.length} items left`}</span>
```

Comment:
maybe here should be the number of left (uncompleted) todos?

Reply:
Done